From 338bdc0bf1ba4f66ac84664ef32f01820b6cac30 Mon Sep 17 00:00:00 2001
From: Hin-Tak Leung <htl10@users.sourceforge.net>
Date: Thu, 21 Nov 2024 04:16:10 +0000
Subject: [PATCH 4/7] skShaper depends on Harfbuzz, and needs a few build
 options

Signed-off-by: Hin-Tak Leung <htl10@users.sourceforge.net>
---
 Source/ThirdParty/skia/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Source/ThirdParty/skia/CMakeLists.txt b/Source/ThirdParty/skia/CMakeLists.txt
index 0f7ffaeb..98080c64 100644
--- a/Source/ThirdParty/skia/CMakeLists.txt
+++ b/Source/ThirdParty/skia/CMakeLists.txt
@@ -975,6 +975,7 @@ endif ()
 
 target_link_libraries(Skia PRIVATE
     Epoxy::Epoxy
+    HarfBuzz::HarfBuzz
     Freetype::Freetype
     JPEG::JPEG
     PNG::PNG
@@ -1028,6 +1029,9 @@ target_compile_definitions(Skia PUBLIC
     SK_DISABLE_LEGACY_PNG_WRITEBUFFER
     SK_DISABLE_TRACING
     SK_DISABLE_WEBGL_INTERFACE
+    SK_SHAPER_PRIMITIVE_AVAILABLE
+    SK_SHAPER_HARFBUZZ_AVAILABLE
+    SK_SHAPER_UNICODE_AVAILABLE
 
     SK_ENABLE_PRECOMPILE
 
-- 
2.47.0

