From e7becf2218b2eeea0e3c04288a8148fbe0692cdf Mon Sep 17 00:00:00 2001
From: Hin-Tak Leung <htl10@users.sourceforge.net>
Date: Thu, 21 Nov 2024 01:15:19 +0000
Subject: [PATCH 5/5] Adding SkShaper to CMakeLists.txt

Signed-off-by: Hin-Tak Leung <htl10@users.sourceforge.net>
---
 Source/ThirdParty/skia/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Source/ThirdParty/skia/CMakeLists.txt b/Source/ThirdParty/skia/CMakeLists.txt
index 581f1f1f..0c467718 100644
--- a/Source/ThirdParty/skia/CMakeLists.txt
+++ b/Source/ThirdParty/skia/CMakeLists.txt
@@ -941,6 +941,12 @@ add_library(Skia STATIC
 
     modules/skresources/src/SkAnimCodecPlayer.cpp
     modules/skresources/src/SkResources.cpp
+
+    modules/skshaper/src/SkShaper.cpp
+    modules/skshaper/src/SkShaper_factory.cpp
+    modules/skshaper/src/SkShaper_harfbuzz.cpp
+    modules/skshaper/src/SkShaper_primitive.cpp
+    modules/skshaper/src/SkShaper_skunicode.cpp
 )
 
 if (ANDROID)
-- 
2.47.0

