From af2e6eb72385df09ce4262a71584dfdb2b0283ed Mon Sep 17 00:00:00 2001
From: Hin-Tak Leung <htl10@users.sourceforge.net>
Date: Fri, 22 Nov 2024 00:02:15 +0000
Subject: [PATCH 6/6] XML code needs expat

Signed-off-by: Hin-Tak Leung <htl10@users.sourceforge.net>
---
 Source/ThirdParty/skia/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/ThirdParty/skia/CMakeLists.txt b/Source/ThirdParty/skia/CMakeLists.txt
index da5707f4..44adc415 100644
--- a/Source/ThirdParty/skia/CMakeLists.txt
+++ b/Source/ThirdParty/skia/CMakeLists.txt
@@ -4,6 +4,7 @@ set_property(DIRECTORY . PROPERTY FOLDER "skia")
 find_package(Freetype 2.9.0 REQUIRED)
 find_package(Fontconfig 2.13.0 REQUIRED)
 find_package(WebP REQUIRED COMPONENTS mux)
+find_package(EXPAT REQUIRED)
 
 if (ANDROID)
     find_package(EXPAT REQUIRED)
@@ -985,6 +986,7 @@ endif ()
 target_link_libraries(Skia PRIVATE
     Epoxy::Epoxy
     HarfBuzz::HarfBuzz
+    EXPAT::EXPAT
     Freetype::Freetype
     JPEG::JPEG
     PNG::PNG
-- 
2.47.0

